Pocket.Logger/Core/Formatter.cs

```diff
@@ -141,36 +145,36 @@ public FormatterResult(StringBuilder formattedMessage)
 
             private readonly List<(string Name, object Value)> properties = new List<(string, object )>();
 
-            public void Add(string key, object value)
-            {
-                properties.Add((key, value));
-            }
+            public void Add(string key, object value) => properties.Add((key, value));
 
-            public IEnumerator<(string Name, object Value)> GetEnumerator()
-            {
-                return properties.GetEnumerator();
-            }
+            public IEnumerator<(string Name, object Value)> GetEnumerator() => properties.GetEnumerator();
```

Comment:
General nit: can you try to make distinct commits for bulk formatting changes?

Reply:
That was the intent of the "R: cleanup" commit. I typically use the R prefix to denote a commit containing only refactoring, which may include formatting changes.